golang/testsuite/testsuite_impl/network_partition_test/network_partition_test_.go

```diff
@@ -113,15 +117,12 @@ func (test NetworkPartitionTest) Run(network networks.Network) error {
 
 	// Adding another API service while the partition is in place ensures that partitiong works even when you add a node
 	logrus.Info("Adding second API container, to ensure adding a network under partition works...")
-	uncastedDatastoreSvc, err := castedNetwork.GetService(datastoreServiceId)
-	if err != nil {
-		return stacktrace.Propagate(err, "An error occurred getting the datastore service interface")
-	}
+
 	apiClient2, err := test.addApiService(
 		castedNetwork,
 		api2ServiceId,
 		apiPartitionId,
-		uncastedDatastoreSvc.(*datastore.DatastoreService))
+		datastoreServiceContext.GetIPAddress(), datastore.Port)
```

Comment:
Let's keep with the established trend of "1 argument per line" here - maybe something like:
```
    datastoreServiceContext.GetIPAddress(), 
    datastore.Port,
)
```

Reply:
Yes, definitely that is better